mkt/constants/applications.py

```diff
@@ -41,6 +49,8 @@ class DEVICE_GAIA(object):
     'mobile': DEVICE_MOBILE.id,
     'tablet': DEVICE_TABLET.id,
     'firefoxos': DEVICE_GAIA.id,
+    'firefoxos+mobile': DEVICE_GAIA.id,
```

Comment:
"firefoxos" should be kept as an alias but not exposed in choices IMHO. Does not necessarily need a migration though, we can convert when instances using the string (Websites, etc) are re-saved.